static/app/views/alerts/list/rules/row.tsx

```diff
@@ -519,4 +519,8 @@ const Label = styled(TextOverflow)`
   margin-left: 6px;
 `;
 
+const PaddedActorAvatar = styled(ActorAvatar)`
+  margin-bottom: 3px;
```

Comment:
It be better if this could be in 2px or 4px. You can use `space(0.25)` or `space(0.5)`. You would also want this margin to apply to the wrapper instead of the individual icon.

Reply:
Hmm - i  chose 3px to add a total of 6px, which makes the 18px of the Default avatar image the same size as the other images (24px). Should we just update the icon sizes to match instead? I tried making the default avatar larger, but it looked a little weird. If i do the 2 or 4px it makes the spacing look at bit weird.

Curious if there's a default component or something like that I could use for both of these images instead of having them be two separate components?

Reply:
Another issue with putting the margin on the PaddedActorAvatar itself is that it adds the margin within the dropdown as well